ui/src/telegrafs/components/Collectors.tsx

```diff
@@ -120,8 +123,8 @@ class Collectors extends PureComponent<Props, State> {
           <Grid.Row>
             <Grid.Column
               widthXS={Columns.Twelve}
-              widthSM={Columns.Eight}
-              widthMD={Columns.Ten}
+              widthSM={collectors.length ? Columns.Eight : Columns.Twelve}
```

Comment:
can you guard here or in the StateProps for collectors being null?

Reply:
@drdelambre like `collectors && collectors.length`? I'll check out if there are other ways of setting a default prop for stateProps to stabilize it on both ends